x/registry/types/did.go

```diff
@@ -117,9 +121,9 @@ func stringifyVerificationRelationships(auth did.VerificationRelationships) ([]s
 	return res, nil
 }
 
-func (d *DIDDocument) DID() did.DID {
+func (d *DIDDocument) DID() *did.DID {
 	id, _ := did.ParseDID(d.Id)
```

Comment:
off-topic, but DIDDocument.Id should probably be DIDDocument.ID to follow not only Go convention, but to be internally consistent as well.
```suggestion
	id, _ := did.ParseDID(d.ID)
```

Reply:
We have a lot of inconsistencies and code cleanup to do after we get our initial launch done. Can you file a bug? Same for the below comment.